### PR TITLE
Release v0.18.4

### PR DESCRIPTION
**Pull request by Stypox**
### New
- Added language selector to change the app language #2921
- Added send to Kodi button to player collapsible menu #2310 
- Added ability to copy comments on long press #3053

### Improved
- Fix ReCaptcha activity and correctly save obtained cookies #3035 
- Remove dot-menu in favour of drawer and hide history button when "Watch history" is not enabled in settings #2960
- Ask for display over other apps permission in settings correctly on Android 6 and later #3004
- Rename local playlist by long-clicking in BookmarkFragment #2954
- Various PeerTube improvements #3044 :
  - Display full description instead of truncated
  - Handle age limit
- Improved several English source strings #2938

### Fixed
- Fixed player starting again although it is paused when option "minimize on app switch" enabled and NewPipe is minimized #2917 
- Fixed `.srt` subtitle downloads containing not all line breaks #2957
- Fixed download to SD card failing because some Android 5 devices are not CTF compliant  #2958
- Fixed downloading on Android KitKat #3032 
- Fixed corrupt video `.mp4` file being recognized as audio file #2959 
- Fixed wrong Chines language codes #2865
- Fix initial brightness value for gesture #3102
- Fixed corrupt mp4 downloads #3117
- [YouTube] Timestamps in description are clickable again #3044

------------------------
## TODO
- [x] Merge #3098
- [x] Merge #3117
- [x] Merge Weblate
- [x] Write Fastlane changelog

------------------------
## APK for testing: RC1
[app-debug_RC1.zip](https://github.com/TeamNewPipe/NewPipe/files/4227084/app-debug.zip)
**PLEASE REPORT _NEW_ CRASHES IN THE CORRESPONDING ISSUE #3114**

### CODE REVIEW

**Comment by TobiGr** [comment]
@poolitzer Do you have time to write a blog post in the next days?

**Comment by Poolitzer** [comment]
Sure thing.

**Comment by Poolitzer** [comment]
can someone explain to me where #2310 got implemented? Cant find it

**Comment by Poolitzer** [comment]
#3102 why is that in improvements and not fixes?

**Comment by Poolitzer** [comment]
#3044 is in there twice. Could be intended.

**Comment by gkeegan** [comment]
@Poolitzer it seems to be intended, since that PR has various changes that I don't think should be condensed to one bullet point.

**Comment by Poolitzer** [comment]
upps.

**Comment by Stypox** [comment]
@Poolitzer fixed about #3102 ;-)

**Comment by Poolitzer** [comment]
Good. I assumed that, so I moved it in the blogpost already (not that it matters this much)

File: app/src/main/res/values-in/strings.xml [hunk 1]
```diff
@@ -525,6 +525,17 @@
...
+    <string name="enable_lock_screen_video_thumbnail_summary">Ketika menggunakan pemutar latar belakang, thumbnail video akan ditampilkan pada layar kunci</string>
+    <string name="default_kiosk_page_summary">Kiosk Default</string>
+    <string name="clear_download_history">Hapus riwayat unduhan</string>
+    <string name="delete_downloaded_files">Hapus berkas yang diunduh</string>
+    <string name="deleted_downloads">%1$s unduhan dihapus</string>
+    <string name="permission_display_over_apps">Izinkan untuk ditampilkan di atas aplikasi lain</string>
+    <string name="app_language_title">Bahasa apl</string>
+    <string name="systems_language">Default sistem</string>
+    <string name="recaptcha_done_button">Selesai</string>
+    <string name="duration_live_button" translatable="false">Langsung</string>
```
Comment (B0pol) [inline app/src/main/res/values-in/strings.xml]:
```suggestion
```

File: app/src/main/res/values-ku/strings.xml [hunk 2]
```diff
@@ -538,4 +537,13 @@
...
     <string name="enable_lock_screen_video_thumbnail_summary">کاتێ کارپێکەری پاشبنەما کاردەکات ئەوا وێنۆچکەی ڤیدیۆکە لە ڕوونما داخراوەکەدا نیشاندەدرێت</string>
+    <string name="clear_download_history">سڕینەوەی مێژووی داگرتن</string>
+    <string name="delete_downloaded_files">سڕینەوەی فایلە داگیراوەکان</string>
+    <string name="deleted_downloads">%1$ لە داگرتنەکان سڕانەوە</string>
+    <string name="permission_display_over_apps">دەسەڵاتی پێبدە بۆ پیشاندان لەسەرووی ئەپەکانیترەوە</string>
+    <string name="app_language_title">زمانی ئەپ</string>
+    <string name="systems_language">بنەڕەتی سیستەم</string>
+    <string name="subtitle_activity_recaptcha">گرتە بکە لەسەر ”تەواو” کاتێ کە چارەسەرکرا</string>
+    <string name="recaptcha_done_button">تەواو</string>
+    <string name="duration_live_button" translatable="false">ڕاستەوخۆ</string>
```
Comment (B0pol) [inline app/src/main/res/values-ku/strings.xml]:
```suggestion
```

File: app/src/main/res/values-pl/strings.xml [hunk 3]
```diff
@@ -539,9 +539,15 @@
...
     <string name="clear_download_history">Wyczyść historię pobierania</string>
     <string name="delete_downloaded_files">Usuń pobrane pliki</string>
-    <string name="deleted_downloads">Usunięte% 1$s pobrania</string>
+    <string name="deleted_downloads">Usunięte %1$s pobrania</string>
+    <string name="permission_display_over_apps">Zezwól na wyświetlanie w innych aplikacjach</string>
+    <string name="app_language_title">Język aplikacji</string>
+    <string name="systems_language">Domyślny systemu</string>
+    <string name="subtitle_activity_recaptcha">Po rozwiązaniu naciśnij „Gotowe”</string>
+    <string name="recaptcha_done_button">Gotowe</string>
+    <string name="duration_live_button" translatable="false">Na żywo</string>
```
Comment (B0pol) [inline app/src/main/res/values-pl/strings.xml]:
```suggestion
```

**Comment by TobiGr** [comment]
@B0pol Thank you. Done
@Stypox Is there anything which prevents a release?

**Comment by Stypox** [comment]
No, I don't think so, we should be good (the apk works and no problem has been reported) :-)

**Comment by TobiGr** [comment]
What the hell is going on. :eyes:  I cannot push to the dev branch. Worked without problems some minutes ago. Now I send endless data to GitHub, but nothing happens.

**Pull request merged**